rtcpeerconnection.go

```diff
@@ -928,18 +927,15 @@ func (pc *RTCPeerConnection) acceptSRTP() {
 		// RTP
 		go func() {
 			for {
-				rtpPacket := &rtp.Packet{}
-				rtpLen, err = r.Read(rtpBuf)
+				rtpBuf := make([]byte, receiveMTU)
```

Comment:
We have to alloc for now because we dupe packets for our user (in existing API)

When we do API breakage we can make strict rules about who owns what data wise